submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/MultipartFileSubmissionApiTest.kt

```diff
@@ -242,6 +244,15 @@ internal class MultipartFileSubmissionApiTest(
             assertThat(savedSubmission["Title"]).isEqualTo("Test Submission")
         }
 
+        @Test
+        fun `invalid format file`() {
+            val submission = tempFolder.createFile("submission.txt")
+
+            assertThatExceptionOfType(WebClientException::class.java)
```

Comment:
I tend to favor here assertThrows which is more kotlin friendly as a parameter as supported as generics. Said that still not big difference

Reply:
Yeah, I like that one better too but it doesn't support inline which we use in the web client, that's why we have to use these in the itests